arcgis-ios-sdk-samples/Content Display Logic/Views/DownloadProgressView.xib

```diff
@@ -33,14 +33,15 @@
                             <nil key="highlightedColor"/>
                         </label>
                         <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Downloading remote resources" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Zjh-ix-0i1">
-                            <rect key="frame" x="8" y="101" width="224" height="21"/>
+                            <rect key="frame" x="8" y="101" width="224" height="41"/>
                             <fontDescription key="fontDescription" type="system" pointSize="17"/>
                             <nil key="textColor"/>
                             <nil key="highlightedColor"/>
                         </label>
                         <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="eOX-4U-fqJ">
-                            <rect key="frame" x="91.5" y="132" width="57" height="33"/>
+                            <rect key="frame" x="91.5" y="152" width="57" height="33"/>
                             <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
+                            <color key="tintColor" name="primaryThemeColor"/>
```

Comment:
Could this be set on the xib's root view so that it would apply to any control in the view, not just this one?